tests/taskcluster/bin/raptor_graph

```diff
@@ -23,6 +23,9 @@ var originalHeadRev = process.env.GITHUB_HEAD_REV;
 
 const TC_CLIENT_OPTS = { timeout: 30 * 1000 };
 
+// taskcluster worker type for raptor tasks
+RAPTOR_WORKER_TYPE = 'raptor-single-capacity';
```

Comment:
Oh I see, ok cool. Yeah we'll leave it as is for now, but I'll look into this more down the road, thanks again